docs/examples/financial/simple_lcoe_calculator.py

```diff
@@ -0,0 +1,67 @@
+"""
+LCOE Calculation
+================
+
+Example of an LCOE calculation using an approach implemented in NREL's "Simple
+LCOE Calculator", accessible at http://www.nrel.gov/analysis/tech-lcoe.html
+"""
+# %%
+# This example shows basic usage of pvlib's lcoe calculation with
+# :py:meth:`pvlib.financial.lcoe` and :py:meth:`pvlib.financial.crf`.
+# The example shown here will generate a Series of annual cost and production
+# data, and a numerical LCOE. To be comparable with NREL's implemenation,
+# this example adheres to the following assumptions: that energy production
+# and O&M costs are constant and that the entire project is financed with a
+# loan. Input values for CAPEX, capacity factor, and O&M were sourced from
+# NREL's ATB for a residential system in 2022 located in Resource Class 5
+# with moderate technological advancement. The discount rate is set to the
+# value recommended in NREL's implementation.
+
+import numpy as np
+import pandas as pd
+from pvlib import financial
+
+n = 20
+
+# Capacity factor
+cf = 0.15357857
+
+# Constant annual energy production
+energy = np.full(n, cf*8760)
+
+# Discount rate
+discount_rate = 0.03
+
+# Debt tenor
+debt_tenor = n
```

Comment:
"Capital recovery period" or "analysis period" would be preferable to "debt tenor" for this variable name. For example, in the ATB we assume an 18 year debt tenor, but then analyze the project for 20 or 30 years. The debt tenor is involved in computing the debt fraction, but doesn't appear explicitly in the FCR formula.

Reply:
This example is not using a FCR method - it is using the procedure outlined in https://www.nrel.gov/analysis/tech-lcoe-documentation.html#:~:text=The%20simple%20levelized%20cost%20of,rate)%20%2B%20variable%20O%26M%20cost. Should I still change the variable name?

Reply:
Ah, thanks for clarifying. Yes, that calculator uses the "analysis period" name:
![image](https://user-images.githubusercontent.com/5530592/223763448-e99fcbaf-3c3d-4c1e-a30b-6d8a1e39e1ab.png)